inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -72,9 +75,11 @@ survey_data$height <- as.numeric(survey_data$height)
 survey_data <- filter(survey_data, age_months > 0, age_months < 60)
 
 survey_data$age_group <- age_categories(survey_data$age_months, breakers = c(6, 18, 30, 42, 54, 60))
```

Comment:
would be good to follow the cholera template cleaning section - breaks it down by variable type